src/style/internal/shared_input-select-picker.scss

```diff
@@ -50,6 +50,17 @@ $height-of-helper-text-pseudo-before: 0.75rem; // There is strange a pseudo befo
     color: $label-color;
 }
 
+@mixin looks-like-input-value() {
+    font-family: Roboto, sans-serif;
+    line-height: functions.pxToRem(28);
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+
+    font-size: functions.pxToRem(14);
+    font-weight: 400;
+    letter-spacing: 0.009375em;
+}
```

Comment:
This is identical to `looks-like-input-label()`, except for the text-colour, since we don't want to link those. I haven't tested this, so we might need to add a colour here too, but that should be the same variable we use for the input _value_ not the variable we use for the _label_ (even if the current _value_ of those variables happen to be the same at the moment).